internal/web/dex_controller.go

```diff
@@ -0,0 +1,257 @@
+package web
```

Comment:
Remove this controller. Traefik forward auth is taking care of these endpoints now.

Reply:
removed